server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/JamesMailSpooler.java

```diff
@@ -107,9 +91,7 @@ public void setMailProcessor(MailProcessor mailProcessor) {
     }
 
     @Override
-    public void configure(HierarchicalConfiguration config) throws ConfigurationException {
-        numDequeueThreads = config.getInt("dequeueThreads", 2);
```

Comment:
Don't we need to document that there is no more "dequeue threads"?
 - Modify `mailetcontainer.xml` to remove this parameter ?
 - Write a note in changelog ?

Reply:
You are right, I'll update the PR after this build, triggering a new build for a doc update is wasteful

Reply:
where is it?

Reply:
I checked the documentation and it doesn't appear anywhere (documentation and sample files)